executor/const.go

```diff
@@ -19,4 +19,6 @@ const (
 	VotePoolQueryMethodName         = "query_vote"
 	VotePoolQueryParameterEventType = "event_type"
 	VotePoolQueryParameterEventHash = "event_hash"
+
+	FinalizedBlockNumber = int64(-3)
```

Comment:
why `-3`

Reply:
it is the parameter to make RPC call for latest finalized block, actually already used the imported constant, so this will bedeleted